index.html

```diff
@@ -42,5 +41,182 @@ <h5>LET'S CONNECT</h5>
                 </div>
             </div>
         </section>
+        <section class="works">
+            <div class="works-flex">
+                <div class="works-flex-item work-image">
+                    <img src="assets/images/portfolio_snapshoot/snapshoot1.png" class="img-style" alt="tonic works" />
+                </div>
+                <div class="works-flex-item">
+                    <div class="intro-text-works">Tonic</div>
+                </div>
+                <ul class=" works-flex-item titles-container">
+                    <li class="list-main">CANOPY</li>
+                    <li class="titles-item">Back End Dev</li>
+                    <li class="titles-item">2015</li>
+                </ul>
+                <div class="works-flex-item works-description">
+                   A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+                <div class="works-flex-item">
+                    <div class="lang-container">
+                        <div class="lang">html</div>
+                        <div class="lang">css</div>
+                        <div class="lang">javascript</div>
+                    </div>
```

Comment:
- [ ] Hi, great work implementing the project technologies section according to the Figma 👏. However, one thing I want to point out is using the correct CSS semantic tags. Since these technologies are technically a list of each technology; in terms of HTML semantics, they should be in a `ul` element with `li` tags for each technology.
something like this would be really nice 👇:
```suggestion
                    <ul class="lang-container">
                        <li class="lang">html</li>
                        <li class="lang">css</li>
                        <li class="lang">javascript</li>
                    </ul>
```

- [ ] Please implement this for all the other project cards as well.